### DISCUSSION TITLE
No video during Nextcloud Talk on Android phones (AIO 7.6.2)

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#3814] [Questions] [answered]
I have observed an issue where video is not working during Nextcloud Talk on Android phones (again for me). The audio is working fine. This issue has been tested on Nextcloud AIO 7.4.2 and has been observed on different Android devices: Samsung Galaxy S21+5G , Huawei Mate 10 Pro and Huawei Mate 20 Lite. 
This issue has been observed on both WiFi and cellular data connections, and it persists even when the Android device is on the same LAN as the Nextcloud AIO server.

Steps to Reproduce:
1.	Launch Nextcloud Talk on an Android device.
2.	Initiate a video call with another user.
3.	Observe that the video feed is not displaying or is extremely choppy.

Expected Behavior:
Video should display during Nextcloud Talk calls on Android devices.

Can any other users confirm this issue? Please share your device model and Nextcloud AIO version if you have experienced this problem.
Additional Notes:
·	The issue does not occur on desktop or web browsers.
·	The issue does not occur on Android phones <-> iPhones video calls
·	The issue seems to be specific to Android devices.

**Comment by szaimen** [comment, marked as answer]
Hi, this is an issue of the talk-android app and should be reported here: https://github.com/nextcloud/talk-android/issues
